Data/Scripts/OreDetectorReforged/PlayerOreDetectorSession.cs

```diff
@@ -39,7 +77,7 @@ public override void UpdateAfterSimulation()
             var settings = TerminalOreDetector.GetStorage(detector);
             var ore = MaterialMappingHelper.Static.naturalOres[o];
             var area = new BoundingSphereD(detector.GetPosition(), settings.range);
-            DetectorServer.Add(new SearchTask(area, ore, settings.count, (results) =>
+            DetectorServer.Add(new SearchTask(area, MyAPIGateway.Session.Player.Character.GetPosition(), ore, settings.count, (results) =>
```

Comment:
The player position is not always relevant

Reply:
what do you mean by that?

Reply:
I mean for eaxmple remote controlling a drone. But it's minor detail dont worry about it.

Reply:
Session.ControlledObject.Entity.GetPosition() - this will work for player and any controlled object

Reply:
Can I make another PullRequest? just discard this one.